CONTRIBUTING.md

```diff
@@ -90,8 +90,37 @@ Go ahead and push your changes to GitHub using this command.
 ```sh
 git push
 ```
-
 #### Developer Certificate of Origin (DCO) Sign off
+=======
+## Cherry-pick PRs to release branches
```

Comment:
I think this `=======` is from a merge conflict?

https://github.com/kyverno/kyverno/commit/f67f145d901c9144dc6ad0c180dad0d9a2fd18e5 changed the DCO section from `##` to `####`, and now there are two such headings.

Reply:
Opened #4192 to address these, I believe.